Assets/BossRoom/Scripts/Client/Game/Character/ClientInputSender.cs

```diff
@@ -161,6 +161,11 @@ void FixedUpdate()
             }
 
             m_ActionRequestCount = 0;
+            
+            if (EventSystem.current.currentSelectedGameObject != null) 
```

Comment:
So this works great for preventing movement. Though there's some actions (namely Archer base attack, mage bolt, mage heal, and rogue's base attack) where it seems like a position under the action UI object is set as the look vector. It would likely be a per action change, as well so not a blocking issue for this PR.